database/1-create-database.sql

```diff
@@ -29,3 +29,12 @@ CREATE TABLE guest (
 	PRIMARY KEY(id)
 )Engine=InnoDB;
 
+CREATE TABLE users (
```

Comment:
Please add the drop sentence for this table to in the top of this file.

Additionally the table must be named `user`, in singular.